tardis/montecarlo/tests/test_packet_source.py

```diff
@@ -75,3 +93,47 @@ def test_bb_packet_sampling(
         assert np.all(np.isclose(nus, ref_df["nus"]))
         assert np.all(np.isclose(mus, ref_df["mus"]))
         assert np.all(np.isclose(unif_energies, ref_df["energies"]))
+
+    def test_bb_packet_sampling_relativistic(
+        self,
+        tardis_ref_data,
+        blackbody_simplesource_relativistic,
+    ):
+        """
+        Parameters
+        ----------
+        tardis_ref_data : pd.HDFStore
+        blackbody_simplesource_relativistic : tardis.montecarlo.packet_source.BlackBodySimpleSourceRelativistic
+        """
+        blackbody_simplesource_relativistic.temperature = 10000
+        blackbody_simplesource_relativistic.beta = 0.25
+
+        nus = blackbody_simplesource_relativistic.create_packet_nus(100)
+        unif_energies = (
+            blackbody_simplesource_relativistic.create_packet_energies(100)
+        )
+        blackbody_simplesource_relativistic._reseed(2508)
+        mus = blackbody_simplesource_relativistic.create_packet_mus(10)
+
+        gamma = np.sqrt(1 - 0.25**2) ** -1
```

Comment:
Should the 0.25 be specified instead as blackbody_simplesource_relativistic.beta? Is there a reason for it to be hard-coded?

Reply:
No specific reason. I changed it now.